include/spdlog/sinks/systemd_sink.h

```diff
@@ -7,10 +7,11 @@
 #include "spdlog/details/null_mutex.h"
 #include "spdlog/details/synchronous_factory.h"
 
-#include <array>
-#include <string>
+#define SD_JOURNAL_SUPPRESS_LOCATION // Don't add code location in sd_journal_send
```

Comment:
what if already defined? please wrap with ifndef

Reply:
I think I should add an `#undef` at the end of the header file, in case `systemd/sd-journal.h` is included somewhere else.
In this case however we want this macro to be defined there at all times. The macro avoids `sd_journal_send` to capture the source location where the function is called (`sink_it_`). Here we want to capture the location where spdlog users place their SPDLOG_LOGGER_* macros, and not where our sink sends the messages to the journal. In other terms, not setting the macro would set additional entry values pointing with  `CODE_FILE=systemd_sink.h` file, `CODE_FUNC=sink_it_` and `CODE_LINE=69`.

Reply:
I found a way to not depend on this macro definition. Hopefully I will have the change soon.